src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -3757,6 +4010,7 @@ namespace HVACUnitarySystem {
                                 AirLoopNumber = AirLoopNum;
                                 AirLoopFound = true;
                                 for (ControlledZoneNum = 1; ControlledZoneNum <= NumOfZones; ++ControlledZoneNum) {
+                                    if (UnitarySystem(UnitarySysNum).ControlZoneNum == 0) continue;
```

Comment:
Cycle here on ControlZoneNum == 0 since a zone can be found when ActualZoneNum = 0 and ControlZoneNum = 0.

This concludes changes actually made for this issue.

Reply:
I don't think this new line is needed. But why is line 4017 setting `.ControlZoneNum` below? That line shouldn't be there, should it?

Reply:
I agree. I put this line here because line 4017 was overwriting UnitarySystem.ControlZoneNum. ControlZoneNum which shouldn't ever be changed after it is read in. I can't recall why this (line 4017) is here. Do no harm. Let me think through this more.

Reply:
I see. This was added? or linked in with? the AT mixer error checking. Return or Supply mixer. I guess this test is only valid for load based control since only that control type requires a control zone name. This condition (load based and control zone = 0) causes a fatal so I guess I should skip all this?